### PR TITLE
Feat:Badges component Added

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Badges component Added
feature of the badge component is added in this component library.

1.Badge on icons is added.
2.Badge on avatars is added.
Please review the code and give your sugggestions to me.

### CODE REVIEW

File: Navigation.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+:root{
+    --primary-color:#1E40AF;
+    --secondary-color:#BFDBFE;
+}
+body{
+    margin: 0;
+    overflow-x: hidden;
```
Comment (Raghav888) [inline Navigation.css:7]: overflow: hidden will also work instead of two property

File: Navigation.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    width: 1rem;
+    top:-1.6rem;
+    left: 2.3rem;
+    color: white;
+}
+.github{
+    color: white;
+    text-decoration: none;
+    margin-top: 1.9 rem;
+    font-size: 19px;
```
Comment (Raghav888) [inline Navigation.css:48]: use rem instead of px

**Review by Raghav888 — APPROVED** [review]
Approved

**Review by tusharkandpal — COMMENTED** [review]
Why on most of the class you're setting position relative ?

**Comment by neetukumari4858** [comment]
> Why on most of the class you're setting position relative ?

Hey tushar,
Because there was need for direction like (top ,left,right ).That's why I have used ,if there is another way for that then tell  me.

**Comment by tusharkandpal** [comment]
> > Why on most of the class you're setting position relative ?
> 
> Hey tushar, Because there was need for direction like (top ,left,right ).That's why I have used ,if there is another way for that then tell me.

well i would suggest to minimize its use ..you can set flexbox properties to your use case... 

it will cause problem while making your site responsive. That's why

**Comment by neetukumari4858** [comment]
> > > Why on most of the class you're setting position relative ?
> > 
> > 
> > Hey tushar, Because there was need for direction like (top ,left,right ).That's why I have used ,if there is another way for that then tell me.
> 
> well i would suggest to minimize its use ..you can set flexbox properties to your use case...
> 
> it will cause problem while making your site responsive. That's why

okey Thx ,
from next time I will try to use flexbox properties for direction instead of  relative position .

**Review by tusharkandpal — APPROVED** [review]
good to go!

**Pull request merged**